include/core/lethe_grid_tools.h

```diff
@@ -18,7 +18,6 @@
 
```

Comment:
lines 16-17, it should be `lethe_grid_tools_h` instead of `lethe_lethegridtools_h`

Reply:
should actually be lethe_lethe_grid_tools_h